packages/ui/src/lib/sidebar/Sidebar.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+	import { getContext } from 'svelte';
+
+	import DefaultSidebarContent from './DefaultSidebarContent.svelte';
+	import SidebarToggle from './SidebarToggle.svelte';
+
+	export let classes = '';
+	export let hideToggle = false;
+
+	const appShell = getContext('appShell');
+	$: isAlignedRight = $appShell.isAlignedRight();
+</script>
+
+<section
+	class="relative bg-core-grey-800 w-full h-full flex flex-col text-white text-sm z-30 {classes}"
+>
+	<div
+		class="absolute"
```

Comment:
```suggestion
		class="absolute dark"
```

I think we need to apply the `dark` class here, or else components like `<Select>` used in the sidebar use the wrong colors